hier_config/utils.py

```diff
@@ -16,7 +50,7 @@ def load_device_config(file_path: str) -> str:
         str: The configuration file contents as a string.
 
     """
-    return Path(file_path).read_text(encoding="utf-8")
+    return _load_file_contents(file_path=file_path)
```

Comment:
This function and `_load_file_contents()` now have the same signature and behavior. Should we consolidate these functions back down to just a single public function? 

Also, `read_text_from_file()` might be a better name for this function.